lib/utils.js

```diff
@@ -0,0 +1,15 @@
+const gitLogParser = require('git-log-parser')
```

Comment:
Perhaps this file could be named `latest-commits`, which exports a `getLatestCommits` directly on `module.exports`? It seems other files are structured that way.

Reply:
The project seems to support both types of `exports` 
ex: 
https://github.com/conventional-changelog/standard-version/blob/3d341c24ba99920aafcefeed3a6bd9d156ea96fb/lib/configuration.js#L11

I'm gonna **hold** on this change for now, since my intentions with this `utils` file was to allow multiple named exports, if more utils should be added in the future people can still use the destruct assignment to get the desired function without having to refactor the default export 
ex: `const {util_1, util_2 } = require('./lib/utils')`

Reply:
What I mean is that the author does have utils separated by files, instead of having a single utils file with all of them there.

Reply:
Got it! 
yes makes sense, I will refactor it